pkg/cmd/cli/cmd/login/logout.go

```diff
@@ -123,25 +126,39 @@ func (o LogoutOptions) RunLogout() error {
 		return err
 	}
 
-	if err := client.OAuthAccessTokens().Delete(token); err != nil {
-		return err
+	errs := []error{}
+	serverErr := client.OAuthAccessTokens().Delete(token)
+	if serverErr != nil {
+		errs = append(errs, serverErr)
+	}
+
+	if configErr := deleteTokenFromConfig(*o.StartingKubeConfig, o.PathOptions, token); configErr != nil {
+		errs = append(errs, configErr)
+	} else if serverErr != nil {
+		// if token was not able to be deleted using api but was successfully removed
+		// from client config, print brief explanation that token was still removed locally
+		fmt.Fprintf(o.ErrOut, "%s\n\n", "An error ocurred deleting the token on the server. The token has been removed from your local configuration.")
```

Comment:
@fabianofranz Thanks, updated this line to `glog` instead. It will still print the server error (if any) however, just in case it is unrelated to the token not being found:

```
$ oc logout --loglevel=1
I0216 09:42:03.042611   13554 logout.go:137] An error ocurred deleting the token on the server. The token has been removed from your local configuration.

Error from server (NotFound): oauthaccesstokens "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9uYW1lc3BhY2UiOiJkZWZhdWx0Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9zZWNyZXQubmFtZSI6ImRlZmF1bHQtdG9rZW4tNjU3bjQiLCJrdWJlcm5ldGVzLmlvL3NlcnZpY2VhY2NvdW50L3NlcnZpY2UtYWNjb3VudC5uYW1lIjoiZGVmYXVsdCIsImt1YmVybmV0ZXMuaW8vc2VydmljZWFjY291bnQvc2VydmljZS1hY2NvdW50LnVpZCI6IjVmMzk0Y2I0LWYzYWYtMTFlNi1iNDUxLTUwN2I5ZGFjOTZlMSIsInN1YiI6InN5c3RlbTpzZXJ2aWNlYWNjb3VudDpkZWZhdWx0OmRlZmF1bHQifQ.c7TiHikwCsobsMTIYhB6O-FjRca6TJmccpN4jem4oKWh9UTE8SxjqnCtrjZe_-zJ0eZ-GovMgiaa5L0xcilcvnj7htKeT3XeycKcUzxYCGJj884UL-Ouu4dh0-vJElvEMu6AlpwxmaES99-aJbghaTOssIIVFQdP9-AsYA2iVS1BBbeCWLw7yrE1gRXVhSjRhntpUwr-OBGO7fV8_ky96FjmUltADw0cynZr6P_NN_DWZYrEhaK7TuAbc-14K9uzLqcAiG_RLJfR7fOUvjPBp9kQDfQ13PczYhvMhXxluMma7WyXOrsB9ia2D2dXJ2SIA4a6tuL_8K_7UPaiO-an9Q" not found
```